src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+  const titleMessage = document.createElement('h2');
+  const descriptionMessage = document.createElement('p');
+
+  message.classList.add('notification');
+  message.classList.add(type);
+  titleMessage.classList.add('title');
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  titleMessage.textContent = title;
+  descriptionMessage.textContent = description;
+
+  document.body.append(message);
+  message.appendChild(titleMessage);
+  message.appendChild(descriptionMessage);
+
+  setTimeout(() => {
+    message.style.visibility = 'hidden';
+  }, '1000');
```

Comment:
Message should be hidden after 2 seconds:
```suggestion
  }, 2000);
```